pkg/handlers/clusterlogging/handler.go

```diff
@@ -43,10 +41,11 @@ func NewHandlers(opts *config.Options) []handlers.RequestHandler {
 	}
 	return []handlers.RequestHandler{
 		&handler{
-			opts,
-			setString{},
-			dm,
-			client,
+			config: opts,
+			whitelisted: setString{},
+			documentManager: dm,
+			osClient: client,
+			projectService: NewRolesService(1000, opts.CacheExpiry, client),
```

Comment:
Why is it a projectService but we create a RoleService?

Reply:
It's just  copy-paste issue I will rename it.